src/pages/Home/mock.ts

```diff
@@ -0,0 +1,37 @@
+export type Subscriptions = {
+  __typename: 'subscriptions';
+  expiration: string;
+};
+
+export type MemberInformation = {
+  __typename: 'members';
+  first_names: string;
+  last_names: string;
+  subscriptions: Subscriptions[];
+};
+
+export type UserType = {
+  __typename: 'users';
+  avatar_url: string | null;
+  username: string;
```

Comment:
Please notice this field is not used anymore

https://did-db.herokuapp.com/console/data/did-db/schema/public/tables/users/modify

You can use `member_code`

Reply:
Oh, thanks for notifying me !